src/Ztm.Zcoin.NBitcoin/Exodus/PropertyAmount.cs

```diff
@@ -132,5 +132,15 @@ public static bool operator>(PropertyAmount first, PropertyAmount second)
         {
             return first.value >= second.value;
         }
+
+        public static PropertyAmount Negate(PropertyAmount amount)
+        {
+            if (amount.value == long.MinValue)
```

Comment:
We don't need this condition. Just wrap `long` negation with `checked` keyword.

Reply:
Cool!